src/devices/SoftPwm/SoftwarePwm.csproj

```diff
@@ -2,12 +2,14 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableDefaultItems>false</EnableDefaultItems>
-    <!--Disabling default items so samples source won't get build by the main library-->
+    <RuntimeIdentifier>linux-arm</RuntimeIdentifier>
```

Comment:
Why do you need to specify  a RuntimeIdentifier here, I think that is wrong.

Reply:
> Why do you need to specify a RuntimeIdentifier here, I think that is wrong.

I thought it looked strange, as well.  I removed and appeared to build locally for binding, but had failures on entire iot build.  Seem to fix it when adding back.  Based it on the RGBLedMatrix project.

Reply:
@joperezr so i just removed and local build was successful.  It must be the following I removed and reason it failed.  I'm assuming that is needed for the Linux thread issue?

I also removed the RuntimeIdentifier from the RGBLedMatrix project (even though this PR was unrelated).

```
<ProjectReference Include="$(MSBuildThisFileDirectory)../../System.Device.Gpio/System.Device.Gpio.csproj">
      <AdditionalProperties>RuntimeIdentifier=linux</AdditionalProperties>
    </ProjectReference>
```